### ISSUE TITLE
Uncaught TypeError: Cannot read property 'childNodes' of undefined

### ISSUE THREAD

**Issue by boussou** [yyx990803/vue#673]
what's wrong with this code?
http://jsfiddle.net/yg1hv747/

I get this error located at function makeChildLinkFn (linkFns)  in **v0.11.4**
**"Uncaught TypeError: Cannot read property 'childNodes' of undefined"**

It don't get it when running it on v10.4

if I put a simpler content like this I got no error
```html
    <div id="test">
    <div class="panel panel-default" v-repeat="list" >
                  {{message}}
              </div>
    </div>
```
I was trying to build an example to isolate the possible memory leak.

It appears it is because of this content in span v-text="options". Why?
```html
            <ul>
              <li>Answer 1</li>
              <li>Answer 2</li>
            </ul>
```

thanks

**Comment by crswll** [comment]
You can put `v-pre` on the `<ul>` and it'll work. I'm not sure why you're getting that error since `document.getElementById('foo').textContent = 'Whatever'` on an element with HTML inside doesn't break, though. I *think* the issue is that it has child nodes when the directive is bound then they're gone after the update but honestly a shot in the dark.

How did you run into this situation? It seems like you'd want `options` to be an array and do `<li v-repeat="options">`. `v-text` does not seem like the right tool for this job.

**Comment by yyx990803** [comment]
This has been fixed by bc254bf942e1f21cc02dda281e5e701c78cb41e8

**Comment by sebek** [comment]
I've run in to this problem as well, where node is undefined.
I had to put a undefined check on node before _.toArray(node.childNodes) to get it working.
